### PR TITLE
JW Player Real Time Data Provider

### PR DESCRIPTION
**Pull request by karimMourra**
## Type of change
- [x] Feature

## Description of change
Adds the jwplayer RTD Provider module which allows Ad Bidders to obtain JW Player's Video Ad Targeting information.
It is a submodule of the RTD Module. 
The module will fetch segments for the media ids present in the prebid config when the module loads. When bid requests are made, it will check for params in the AdUnit set by the publisher and will set the corresponding targeting information to the real time data returned in `getData`.

- contact email of the adapter’s maintainer
karim@jwplayer.com

## Other information
The base branch of this PR is #5519 and should be merged after it. The only changes made to it are the addition of `jwplayerRtdProvider.js` and `jwplayerRtdProvider_spec.js`, and the modification of `.submodules.json`

### CODE REVIEW

**Comment by patmmccann** [comment]
I could be crazy but I dont see 'registerSubModule' in the jw module js.

File: modules/jwplayerRtdProvider.js [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+  const params = jwplayerProvider && jwplayerProvider.params;
+  if (!params) {
+    return;
+  }
+  const rtdModuleTimeout = params.auctionDelay || params.timeout;
+  requestTimeout = rtdModuleTimeout === undefined ? requestTimeout : Math.max(rtdModuleTimeout - 1, 0);
+  fetchTargetingInformation(params);
+});
+
+submodule('realTimeData', jwplayerSubmodule);
```
Comment (karimMourra) [inline modules/jwplayerRtdProvider.js:53]: @patmmccann this is the line where we register as an RTD submodule

Comment (patmmccann) [reply modules/jwplayerRtdProvider.js:53]: ty! how do you like this format?

Comment (karimMourra) [reply modules/jwplayerRtdProvider.js:53]: Do you mean how do I like having the jwplayer module be a submodule of an existing module ? I think it's great for reusability and consistency across all rtd providers. Thank you for suggesting this approach.

**Comment by karimMourra** [comment]
@idettman the PR that was blocking this PR has been merged.  Could  you please review ?

**Comment by karimMourra** [comment]
@robertrmartinez Could you please  review this PR ? It was blocked by #5519

**Pull request merged**